package.json

```diff
@@ -36,6 +36,7 @@
     "spritesheet:system-action": "spritesheet-js -f json -p src/assets/images/spritesheets/ --padding 8 --divisibleByTwo -n sprite-system-action-spritesheet --powerOfTwo src/assets/images/sprites/action/*",
     "spritesheet:system-notice": "spritesheet-js -f json -p src/assets/images/spritesheets/ --padding 8 --divisibleByTwo -n sprite-system-notice-spritesheet --powerOfTwo src/assets/images/sprites/notice/*",
     "storybook": "storybook dev -p 6006",
+    "build-storybook": "storybook build",
```

Comment:
Looking at the way the `storybook-to-ghpages` from **_storybook-deployer_** behaves, it first looks for "`build-storybook`" in scripts, else it runs `build-storybook` as is. Storybook 7, the script has changed slightly to `storybook build`, which has been updated here.